tests/fakeapi/hello/__init__.py

```diff
@@ -266,11 +266,11 @@ def test_schema_int(test_int):
 
 
 def test_get_someint(someint):
-    return type(someint).__name__
+    return '%s %g' % (type(someint).__name__, someint)
```

Comment:
Please use "new" Python f-strings (`f"{x} {y}")

Reply:
Sure. When I submitted this PR 9 months ago I reviewed your code style and there were no f-strings around so I carefully converted all my f-strings into old-school % formats.

Reply:
True, we are still catching up with Python features and have now removed some of the old-school stuff (probably a lot of cleaning up still needed).

Reply:
Yeah, been there.